tool/src/main.rs

```diff
@@ -384,6 +392,33 @@ fn main() {
                     process::exit(1);
                 },
             },
+            "keymap" => {
+                let layer = parse_arg(&mut args, "layer");
+                let output = parse_arg(&mut args, "output");
+                let input = parse_arg(&mut args, "input");
+                match args.next() {
+                    Some(value_str) => match u16::from_str_radix(&value_str, 16) {
```

Comment:
It will be confusing that `value` is parsed as hex but a `0x` prefix cannot be used.

Reply:
0x prefix trimmed in https://github.com/system76/ec/pull/97/commits/008ee25785e515f08dbd4be04a25a940399c53d4